packages/core/modules-sdk/src/medusa-app.ts

```diff
@@ -402,10 +403,34 @@ async function MedusaApp_({
     return await remoteQuery.query(query, variables, options)
   }
 
-  const applyMigration = async (linkModuleOptions, revert = false) => {
-    for (const moduleName of Object.keys(allModules)) {
-      const moduleResolution = MedusaModule.getModuleResolutions(moduleName)
+  const applyMigration = async ({
```

Comment:
suggestion: As we discussed, we can probably move this one outside the main function wdyt?

Reply:
Uses `sharedContainer` and `sharedConfig` from the upper scope. Should we pass them down as parameters?

Reply:
I think so, just to lighten this function a bit to simplify reading and understanding